std-bits/table/src/main/java/org/enso/table/data/column/storage/numeric/LongStorage.java

```diff
@@ -198,4 +187,70 @@ public LongStorage widen(IntegerType widerType) {
     assert widerType.fits(getType());
     return new LongStorage(data, (int) getSize(), getIsNothingMap(), widerType);
   }
+
+  /** Allow access to the underlying data array for copying. */
+  public long[] getArray() {
```

Comment:
This should be an internal methods, not something exposed to everyone. This way we are just asking clueless developers to call this method.